src/core/client/admin/components/UserStatus/BanModal.tsx

```diff
@@ -98,8 +98,16 @@ const BanModal: FunctionComponent<Props> = ({
     viewerScopes.role === GQLUSER_ROLE.MODERATOR &&
     !!(!viewerScopes.sites || viewerScopes.sites?.length === 0);
 
+  const userIsBlanketBanned = !!userBanStatus?.active;
```

Comment:
This works fine, but since I don't think we use the term `blanket banned` anywhere else, I would lean toward making this something more like `userIsAllSitesBanned`.